Sugar.Data/Sugar.Data.Nougat.OSX.oxygene

```diff
@@ -12,7 +12,7 @@
     <AssemblyName>Sugar.Data</AssemblyName>
     <DefaultUses>Foundation</DefaultUses>
     <StartupClass />
-    <DeploymentTargetVersion>10.6</DeploymentTargetVersion>

```

Comment:
As discussed via e-mail, NSRegularExpression only appeared in version 10.7 of OS X, and @dwarfland gave the green light to bump up the minimum targeted version.